tools/clang/test/HLSLFileCheck/d3dreflect/rayquery-reflection.hlsl

```diff
@@ -0,0 +1,14 @@
+// RUN: %dxilver 1.9 | %dxc -T lib_6_9 -validator-version 1.9 %s | FileCheck -check-prefix=NOD3D %s
+// RUN: %dxilver 1.9 | %dxc -T lib_6_9 -validator-version 1.9 %s | %D3DReflect %s | FileCheck -check-prefix=D3D %s
+
+// NOD3D: ; RaytracingPipelineConfig1 rpc = { MaxTraceRecursionDepth = 32, Flags = RAYTRACING_PIPELINE_FLAG_SKIP_TRIANGLES | RAYTRACING_PIPELINE_FLAG_ALLOW_OPACITY_MICROMAPS };
+// D3D: MinShaderTarget: 0x10069
```

Comment:
Note: this `MinShaderTarget: 0x10069` is based only on the use of the `AllocateRayQuery2` DXIL op in the shader function.  It should have nothing to do with the subobject flag being introduced in this PR.